webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -13,13 +13,29 @@
 {% javascript 'bugzilla' %}
 {% javascript 'correlation' %}
 {% javascript 'report_index' %}
+<script type="text/javascript">
+    var SocReport = {
+        base: '{{ url('crashstats:correlations_json') }}',
+        path: '{{ make_query_string(product=product, version=report.version, platform=report.os_name, signature=report.signature) }}',
+        loading: 'Loading <img src="{{ static('img/loading.png ') }}" width="16" height="17" alt="Loading..." />'
+    };
```

Comment:
I thought you were in favor of using data attributes instead of inline JS to avoid XSS vulnerabilities as much as possible? :)